### ISSUE TITLE
Compatibility with TypeDoc 0.26.x

### ISSUE THREAD

**Issue by willeastcott** [Drarig29/typedoc-plugin-extras#22]
[TypeDoc 0.26.0](https://github.com/TypeStrong/typedoc/releases/tag/v0.26.0) was released over the weekend. `typedoc-plugin-extras` is not currently compatible with it.

Sure, [this peerDependecy](https://github.com/Drarig29/typedoc-plugin-extras/blob/main/package.json#L37) needs updating, but there seem to be other problems regarding calling the TypeDoc API that need fixing.

Building the [PlayCanvas API reference manual](https://github.com/playcanvas/api-reference) gives:

```bash
C:\dev\api-reference>npm run build

> @playcanvas/api-reference@1.0.0 build
> typedoc

[info] Loaded plugin C:\dev\api-reference\typedoc-plugin-property.mjs
[error] The plugin typedoc-plugin-extras could not be loaded
[error] TypeError: app.renderer.once is not a function
    at load (C:\dev\api-reference\node_modules\typedoc-plugin-extras\dist\main.js:80:18)
    at loadPlugins (C:\dev\api-reference\node_modules\typedoc\dist\lib\utils\plugins.js:30:23)
    at async Application.bootstrapWithPlugins (C:\dev\api-reference\node_modules\typedoc\dist\lib\application.js:222:13)
    at async main (C:\dev\api-reference\node_modules\typedoc\dist\lib\cli.js:42:15)
[info] Loaded plugin typedoc-plugin-mdn-links
[info] Loaded plugin typedoc-plugin-rename-defaults
[info] Loaded plugin typedoc-plugin-missing-exports
[error] Found 2 errors and 0 warnings
```

**Comment by Drarig29** [comment]
Hi @willeastcott! Sorry I didn't see the issue... I'm working on a new release.

**Comment by Drarig29** [comment]
I just published [v3.1.0](https://github.com/Drarig29/typedoc-plugin-extras/releases/tag/v3.1.0).
